docs/components/Buttons.md

```diff
@@ -65,38 +67,36 @@ style `@style/Widget.Orange.Button.Text` on your `Button` layout
 In the layout:
 
 ```xml
-<Button android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Text button"
-        style="@style/Widget.Orange.Button.Text" />
+
```

Comment:
There is an extra line here and in other `xml` blocks below.

Reply:
Hum. I'm not able to remove the blank lines in the XML blocks... My formatter add a blank line at the beginning of each block but I don't know why...